### DISCUSSION TITLE
Add polygon by iD URL arguments

### DISCUSSION THREAD

**Discussion by JackNUMBER** [openstreetmap/iD#9084] [Q&A]
Hello, I'm looking for a documentation about iD URL parameters for GET request. I searched Google and OSM Wiki without result.
My goal is to add a "virtual" polygon on map as boundaries for the contributor (like HOT does with its integrated iD).

eg. `https://www.openstreetmap.org/edit?editor=id#map=2/24.1/8.6&comment=#default message&hashtags=#hashOne,#hashTwo` has `map`, `hashtags`, `comment` arguments

Thank you

**Comment by i-ky** [comment]
I had the same question and my research shows that HOT tasking-manager provides boundaries as GPX file via `gpx` URL parameter:
https://github.com/hotosm/tasking-manager/blob/c255f34c59ece429f1aa2fa99c3097a42a9d7252/backend/services/mapping_service.py#L212
https://github.com/hotosm/tasking-manager/blob/c255f34c59ece429f1aa2fa99c3097a42a9d7252/frontend/src/components/editor.js#L85
